demo/start.sh

```diff
@@ -1,24 +1,29 @@
 #!/usr/bin/env bash
 
+# Disable unicode.
+LC_ALL=C
+LANG=C
+
 DIR="$( cd "$( dirname "$0" )" && pwd )"
 
 CWD=$(pwd)
 
-cd "$DIR/.."
+cd "$DIR/.." || exit
```

Comment:
I note that you have gone through and added `|| exit` to commands that change directory (`cd`, `pushd`, `popd`). It's true that these scripts could do with more error-checking, but this approach seems strange. These are far from the only commands that can fail - indeed, they are generally the least likely to fail. (`cd $(pwd)/..` can only fail in the most unusual circumstances). Why did you pick out these particular commands for error-checking? Wouldn't it make more sense just to put `set -e` at the top of the scripts?